operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainStatus.java

```diff
@@ -275,11 +275,12 @@ public DomainStatus withReplicas(Integer replicas) {
   /**
    * Status of WebLogic servers in this domain.
    *
-   * @return servers
+   * @return servers sorted list of ServerStatus containing status of WebLogic servers in this domain
```

Comment:
nitpick: you don't name the return value (which is what i assume "servers") is, you just explain it